packages/app/src/lib/components/header.svelte

```diff
@@ -62,8 +66,16 @@
 			</div>
 		</section>
 		<div class="flex items-center gap-4">
-			<Button variant="primary" class="sm:flex p-2 relative" href="/dash/messages">
-				<span class="i-tdesign-mail text-black w-6 h-6"></span>
+			{#if $ndkStore.activeUser}
+				<a href={`/p/${$ndkStore.activeUser.profile?.nip05 ? $ndkStore.activeUser.profile?.nip05 : $ndkStore.activeUser.pubkey}`}>
+					<Avatar class="rounded-md border-secondary border-2">
```

Comment:
to not break the current design in other places, i copied the Avatar component and brought it here with some changes